packages/designsystem/package.json

```diff
@@ -1,23 +1,28 @@
 {
   "name": "@locokit/designsystem",
-  "version": "0.8.0",
+  "version": "1.0.0-alpha.0",
   "description": "LocoKit Design System, components",
-  "main": "src/index.ts",
+  "author": "Mathieu Dartigues",
+  "license": "MIT",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.es.js",
+  "types": "dist/src/index.d.ts",
   "scripts": {
     "lint": "eslint .",
-    "story:dev": "histoire dev",
+    "dev": "histoire dev",
     "story:build": "histoire build",
-    "story:preview": "histoire preview"
+    "story:preview": "histoire preview",
+    "build": "rm -rf dist && vite build"
```

Comment:
Can we remove "story:build" to avoid confusion?

Reply:
Mmm, this is not the same goal ?
`story:build` build the "histoire"book,
`build` build the design system package.

I prefer keep them both ?

Reply:
ok